UPGRADE.md

```diff
@@ -2666,6 +2673,24 @@ const ReviewEditToolbar = (props: ToolbarProps<Review>) => {
 };
 ```
 
+Since the `<Toolbar>` component is not cloned anymore by react-admin layouts, properties like `mutationMode` and `submitOnEnter` are not passed down from parent components. So, if you rely on this behaviour, you should create your own custom toolbar instead
+
+```diff
+import { Toolbar, SimpleForm, Edit, TextInput } from 'react-admin';
+
++const MyToolbar = props => <Toolbar {...props} mutationMode="pessimistic" submitOnEnter={false} />;
+
+export const PostEdit = (props) => (
+-    <Edit {...props} mutationMode="pessimistic">
```

Comment:
`mutationMode` is used by `useEditController`, so is valid to pass it as a prop of an `Edit`/`EditBase` but it won't get to the toolbar's `DeleteButton`.